.eslintrc

```diff
@@ -16,19 +16,12 @@
     "quotes": [2, "single", "avoid-escape"],
     "strict": [2, "never"],
     "semi-spacing": [2, {"before": false, "after": true}],
-    "react/jsx-uses-react": 2,
```

Comment:
Do the tests work if you remove this? I tried to remove it but all the tests failed for me.